cscareerhub/megabot
# Feature/message parser

### Purpose of pull request

**Related Issue**: N/A

Add event emitter to dispatch messages to various sections of the bot.

### Pull request checklist


### Testing instructions

Run the bot and try a command with a sub command like `++pins list`. The logger should log the sub command if it gets passed to the pins' handler.
